src/main/java/hw11/PersonFilter.java

```diff
@@ -0,0 +1,26 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class PersonFilter {
+    public static void main(String[] args) {
+        List<Person> persons = new ArrayList<>();
```

Comment:
Code isn't compiled
![image](https://github.com/user-attachments/assets/1036c62c-c112-4984-9ac5-769ccf1062b2)

So fix it and push code only after your success running